src/sagemaker/workflow/steps.py

```diff
@@ -93,6 +93,26 @@ def ref(self) -> Dict[str, str]:
         return {"Name": self.name}
 
 
+@attr.s
+class CacheConfig:
+    """Step to cache pipeline workflow.
+
```

Comment:
Can you add a bit more description to what is cached, how caching affects subsequent executions, and what expiration effectively means?

Reply:
I'll add more description. Expiration definition is added in the attributes